Taggers/python/dumperConfigTools.py

```diff
@@ -7,14 +7,17 @@ def addCategories(pset,cats,variables,histograms,mvas=None):
         addCategory(pset,*cat,variables=variables,histograms=histograms,mvas=mvas)
 
 # -----------------------------------------------------------------------
-def addCategory(pset,label,cutbased=None,subcats=0,variables=[],histograms=[],mvas=None,systLabel=""):
+def addCategory(pset,label,cutbased=None,subcats=0,variables=[],histograms=[],mvas=None,systLabel="",classname=""):
     
+   
     if subcats >= 0:
         catDef = cms.PSet(label=cms.string(label),
+				                  cutbased=cms.bool(cutbased!=None),
                           subcats=cms.int32(subcats),
                           variables=cms.VPSet(),
                           histograms=cms.VPSet(),
-													systLabel=cms.string(systLabel)
+													systLabel=cms.string(systLabel),
+													classname=cms.string(classname)
```

Comment:
Please set the default classname to None and replace the line above with.
if classname: catDef.classname = cms.string(classname).

And similarly for systlabel.